tests/consensus_tests/test_routing_token.py

```
@@ -0,0 +1,158 @@
+    r = requests.post(
+        f"{peer_uri}/collections/{COLLECTION_NAME}/points",
+        json={"ids": [POINT_ID], "with_payload": True},
+        headers=headers,
+    )
+    assert_http_ok(r)
+    points = r.json()["result"]
+    return points[0]["payload"]["marker"] if points else None
+
+
+def create_snapshot(peer_uri):
+    r = requests.post(f"{peer_uri}/collections/{COLLECTION_NAME}/snapshots?wait=true")
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)